config/airkeeper.json.example

```diff
@@ -0,0 +1,21 @@
+{
+  "chains": [
```

Comment:
I assume the chains are matched to the respective chain in airnode.json via the same index? I think it would be a good idea to specify the chainId here as well

Reply:
This is a really good point. I'm not sure how merge does the matching but I'll test it with multiple chains in the config.

Reply:
Indeed it's indexed based. I would really like to have the RrpBeaconServer contract address in the Airnode config.json and get rid of this config in Airkeeper. This would make it a lot less error prone in my opinion.